tests/Unit/Product/SyncerHooksTest.php

```diff
@@ -0,0 +1,283 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\Product;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\DeleteProducts;
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\JobRepository;
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\UpdateProducts;
+use Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter\MerchantCenterService;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\BatchProductHelper;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\ProductHelper;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\SyncerHooks;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\WC;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\ContainerAwareUnitTest;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Tools\HelperTrait\ProductTrait;
+use PHPUnit\Framework\MockObject\MockObject;
+use WC_Helper_Product;
+
+/**
+ * Class SyncerHooksTest
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\Product
+ *
+ * @property MockObject|MerchantCenterService $merchant_center
+ * @property BatchProductHelper               $batch_helper
+ * @property ProductHelper                    $product_helper
+ * @property MockObject|JobRepository         $job_repository
+ * @property MockObject|UpdateProducts        $update_products_job
+ * @property MockObject|DeleteProducts        $delete_products_job
+ * @property WC                               $wc
+ * @property SyncerHooks                      $syncer_hooks
+ */
+class SyncerHooksTest extends ContainerAwareUnitTest {
+
+	use ProductTrait;
+
+	public function test_create_new_simple_product_schedules_update_job() {
+		$this->update_products_job->expects( $this->once() )
+								  ->method( 'start' );
```

Comment:
I'm getting warnings here, should this be renamed to `schedule`?

Reply:
> I'm getting warnings when running the tests:
> 
> ```
> There were 17 warnings:
> 
> 1) Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\Product\SyncerHooksTest::test_create_new_simple_product_schedules_update_job
> Trying to configure method "start" which cannot be configured because it does not exist, has not been specified, is final, or is static
> ```
> 
> I think it has to do with the merge of the function `start` changing to `schedule`.

Sorry missed a bunch of these. All renamed in https://github.com/woocommerce/google-listings-and-ads/pull/862/commits/bbf011f3f667eac2a452a8b6fbb5bc842ddf2984